shell/common/switches.h

```diff
@@ -165,6 +165,10 @@ DEF_SWITCH(DisableDartAsserts,
            "disabled. This flag may be specified if the user wishes to run "
            "with assertions disabled in the debug product mode (i.e. with JIT "
            "or DBC).")
+DEF_SWITCH(ForceMultithreading,
+           "force-multithreading",
+           "Forcibly turn on multithreading. Currently only used in the "
```

Comment:
"Uses separate threads for the platform, UI, GPU and IO task runners. By default, a single thread is used for all task runner. Only available on in the flutter_tester." or something similar may be more descriptive. The current version reads a bit vague without context IMO.